generator/model.go

```diff
@@ -2059,23 +2059,22 @@ func (sg *schemaGenContext) makeGenSchema() error {
 			sg.GenSchema.ExtraImports[alias] = pkg
 		}
 
-		if tpe.IsEmbedded {
-			sg.GenSchema.IsAnonymous = true
```

Comment:
doesn't this change the behavior

Reply:
yes, it should revert #3122 as I found out that this was the change that broke the tests and introduced issue #3142

Reply:
I think in this case it's easier to revert #3122 for now to make the master branch stable again, and then create a new PR for the #3122 feature (revert the revert) with the setup where the CI works well and triggers a failure on test errors and try to fix it in this new PR

Let me know what you think